FlorenceBML/BMLacq38.xml

```diff
@@ -60,59 +60,61 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     </explicit>
                         </msItem>
                         <msItem xml:id="ms_i3">
-                            <locus from="118" to="120r"/>
-                            <title>Chronology from Adam until Ləbna Dəngəl</title>
-                            <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ። ወእምዝ፡ ንጽሕፍ፡ በረድኤተ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ዘንተ፡ መጽሐፈ፡ ሐሰብ። 
-                                እም፡ ፍጥረተ፡ ዓለም፡ እስከ፡ ሕልቀተ፡ ዓለም። እም፡ አዳም፡ እስከ፡ ልደተ፡ ኖኅ፡ <date calendar="creation">፲፻ወ፯፻፡፶ወ፯።</date> እምልደተ፡ ኖኅ፡ እስከ፡ አይኅ፡ 
-                                <date calendar="creation">፯፻።</date></incipit>
-                            <explicit xml:lang="gez">እምዘርአ፡ ያዕቆብ፡ እስከ፡ ብእደ፡ ማርያም፡ ፴ወ፬። ወኮነ፡ ድሙር፡ ፷፻ወ፭፻፡ ወ፭። እምብእደ፡ ማርያም፡ እስከ፡ ውናግ፡ ሰገደ፡ ዘተሰምየ፡ ልብነ፡ 
-                                ድንግል፡ ፴ወ፱። ወኮነ፡ ድሙር። ፷፻ወ፭፻፡ ፵ወ፬። እም፡ ውናግ፡ ሰገደ፡ ዘተሰምየ፡ <persName ref="PRS6229LebnaDe">ልብነ፡ ድንግል። </persName></explicit>
+                             <msItem xml:id="ms_i3.1">
```

Comment:
something like `<title>Chronographcal notes</title> `for ms_i3

Reply:
Done. I hesitated to give a title, because some item are more calendaric (Chronological Computus relating to epact and maṭqəʿ) or astronomic (Note on the moon phases). If we agree, that chronographic (rather than chronologic) may be suitable as umbrella term for all of them, I can give that title to ms_i3.